dev.Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Local tooling webserver wrapper for Node.js runner
+#
+# - first build our gem binaries
+# - then copy them on top of our base test-runnner
+
+#############
+## Stage 1 ##
+#############
+FROM node:erbium-buster as gembuilder
+
+# Install Ruby
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y ruby2.5 ruby-dev;
+
+# Install the gems for the Ruby webserver
+RUN gem install -N --no-wrappers exercism-local-tooling-webserver
+
+#############
+## Stage 2 ##
+#############
+FROM exercism/javascript-test-runner-base
+
+USER root
+
+# Install Ruby
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y ruby2.5; \
```

Comment:
```suggestion
    apt-get install -y ruby2.5; \
```

Shouldn't we use 2.6 as that's what Jeremy is using? Or is that not available for erbium-buster?

Reply:
Not trivially available and it's really irrelevant just for a tiny web wrapper... I'd rather use the built-in Ruby than put a bunch of effort into running a specific version a 50 line Ruby Sinatra app.

Reply:
Agreed. I'm okay with bundling whatever the system has. Wouldn't `ruby` work or must it be `ruby2.5`?

Reply:
Re-producable builds, no? ;-)

Reply:
On buster it will never go `ruby2.6`!

Reply:
I like being explicit - at a glance I know what is installed.

Reply:
FWIW, Josh is removing the Ruby bit entirely and replacing it with an executable.